### PR TITLE
Remove trusted property from Addon.  Also admin_review_type, mozilla_contact

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

**Comment by eviljeff** [comment]
fwiw, mozilla_contact was a Marketplace thing that was never used for addons.

File: src/olympia/devhub/tests/test_views.py [hunk 1]
```diff
@@ -3108,13 +3102,13 @@ def test_add_multi_file_beta(self):
     def test_force_not_beta(self):
         self.post(beta=False)
         f = File.objects.latest()
-        assert f.status == amo.STATUS_PUBLIC
+        assert f.status == amo.STATUS_UNREVIEWED
```
Comment (eviljeff) [inline src/olympia/devhub/tests/test_views.py]: This test was using a trusted fixture so it expected to go straight to PUBLIC

File: src/olympia/signing/tests/test_views.py [hunk 2]
```diff
@@ -237,7 +237,7 @@ def test_version_is_beta_unlisted(self):
         version = qs.get()
         assert version.addon.guid == self.guid
         assert version.version == version_string
-        assert version.statuses[0][1] == amo.STATUS_LITE
+        assert version.statuses[0][1] == amo.STATUS_UNREVIEWED
```
Comment (tofumatt) [inline src/olympia/signing/tests/test_views.py]: Why does this go from `STATUS_LITE` to `STATUS_REVIEWED`?

Comment (eviljeff) [inline src/olympia/signing/tests/test_views.py]: This was another test that was using a trusted fixture (addon_3615).  Though tbh I wasn't doing addons when signing was added so I wasn't 100% what the flow was supposed to be generally for unlisted addons.

**Comment by tofumatt** [comment]
I'm not sure about https://github.com/mozilla/addons-server/pull/2672/files/bebbf1f96e2cf6c8c301be4a3dde24919517e09d#r63368252 but am otherwise r+

**Comment by diox** [comment]
r+ from me too, like @eviljeff said those tests were using a trusted add-on so it was messing things up.

**Pull request merged**